project_milestone/tests/test_project_milestone.py

```diff
@@ -65,3 +65,11 @@ def test_sub_task(self):
         with Form(Task) as task:
             task.project_id = self.test_project
             task.name = "SubTask"
+
+    def test_copy_project(self):
+        project = self.test_project.copy({})
+        tasks = project.with_context(active_test=False).task_ids
+        milestone = project.milestone_ids.filtered(
+            lambda milestone: "2" not in milestone.name
+        )
+        assert tasks[0].milestone_id == milestone and tasks[1].milestone_id == milestone
```

Comment:
chore (non-blocking): would prefer the usage of `self.assertTrue` or similar instead of the raw `assert`